### PR TITLE
[ISSUE-420] Update volumes annotations on disk replacement

### PR DESCRIPTION
**Pull request by mishoyama**
### Issue #420 

CSI must put `replacement: ready`  annotation on the volume during disk replacement to inform application Operator that PVC/PV can be removed.

## PR checklist
- [x] Add link to the issue
- [x] Choose PR label
- [ ] New unit tests added
- [x] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved

## Testing
Tested DR flow manually:
```
borism1@ubuntu:/workspace/csi-baremetal$ volumes
Namespace   Id                                         Size        Type   Health   OpStatus    CSIStatus   Usage      Location                               Node
mishoyama   pvc-e2b465dd-6677-42ec-a5b3-f10600e0cb9a   210763776   xfs    BAD      OPERATIVE   PUBLISHED   RELEASED   9edc2e2f-02ec-4cda-b6ef-c3e7ce26bb3b   5dfbb668-3397-403e-a09a-bc4957508c77
...
borism1@ubuntu:/workspace/csi-baremetal$ kubectl annotate drive 9edc2e2f-02ec-4cda-b6ef-c3e7ce26bb3b replacement=ready
drive.csi-baremetal.dell.com/9edc2e2f-02ec-4cda-b6ef-c3e7ce26bb3b annotated

borism1@ubuntu:/workspace/csi-baremetal$ drives
Id                                     Size        Type   SN                   Health   Usage      Status   System   Node
9edc2e2f-02ec-4cda-b6ef-c3e7ce26bb3b   210763776   HDD    LOOPBACK3138237561   BAD      REMOVING   ONLINE   <none>   5dfbb668-3397-403e-a09a-bc4957508c77

borism1@ubuntu:/workspace/csi-baremetal$ kubectl describe volume pvc-e2b465dd-6677-42ec-a5b3-f10600e0cb9a
Name:         pvc-e2b465dd-6677-42ec-a5b3-f10600e0cb9a
Namespace:    mishoyama
Labels:       <none>
Annotations:  release: done
              replacement: ready
```

### CODE REVIEW

**Review by safronovD — APPROVED** [review]

File: pkg/crcontrollers/drive/drivecontroller.go [hunk 1]
```diff
@@ -33,6 +33,12 @@ type Controller struct {
 	log            *logrus.Entry
 }
 
+const (
+	ignore uint8 = 0
```
Comment (safronovD) [inline pkg/crcontrollers/drive/drivecontroller.go:37]: Is that types of standard return codes from Reconcile?

Comment (mishoyama) [reply pkg/crcontrollers/drive/drivecontroller.go:37]: No, these constants just to make code more readable and update/delete drive CR in one place.

Comment (safronovD) [reply pkg/crcontrollers/drive/drivecontroller.go:37]: Got it, thanks

**Review by hoklet — APPROVED** [review]

**Comment by mishoyama** [comment]
Custom CI passed.

**Pull request merged**